pkg/detectors/falsepositives.go

```diff
@@ -90,3 +91,41 @@ func bytesToCleanWordList(data []byte) []string {
 	}
 	return words
 }
+
+// DefaultMinEntropy setting inspired by the original TruffleHog's default value for strings
+const DefaultMinEntropy = 3.0
+
+func StringShannonEntropy(input string) float64 {
```

Comment:
suggestion: If pre-compute the inverse of the length we can use multiplication instead of division within the entropy calculation.
ex:
```
func StringShannonEntropy(input string) float64 {
	chars := make(map[rune]float64)
	inverseTotal := 1 / float64(len(input)) // precompute the inverse

	for _, char := range input {
		chars[char]++
	}

	entropy := 0.0
	for _, count := range chars {
		probability := count * inverseTotal
		entropy += probability * math.Log2(probability)
	}

	return -entropy
}
```

This gives us a nice ~40% perf gain:
![Screenshot 2023-10-08 at 11 08 41 AM](https://github.com/trufflesecurity/trufflehog/assets/21311841/d8981c71-1e1f-4d45-aab4-f5c81f0ebef3)

Reply:
Oh nice. Added test and changed it out.